hookman/plugin_config.py

```diff
@@ -10,6 +10,18 @@
 from hookman.exceptions import SharedLibraryNotFoundError
 from hookman.hookman_utils import load_shared_lib
 
+from strictyaml import Map, MapPattern, Optional, Str
+
+PLUGIN_CONFIG_SCHEMA = Map(
+    {
+        "caption": Str(),
+        "version": Str(),
+        "author": Str(),
+        "email": Str(),
+        "id": Str(),
+        Optional("extras"): MapPattern(Str(), Str()),
+    }
+)
 
 @attr.s(frozen=True)
```

Comment:
Unfortunately I don't think this can be frozen anymore given that `extras` is a mutable dict now.